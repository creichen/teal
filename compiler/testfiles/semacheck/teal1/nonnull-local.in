fun main() = {
    var x : nonnull int;
    x := 5;
    return x;
}