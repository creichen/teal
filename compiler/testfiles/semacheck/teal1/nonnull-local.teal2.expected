testfiles/semacheck/teal1/nonnull-local.in[2:5-2:24]: Qualified local variabes must be initialized.
